pybikes/data/callabike.json

```diff
@@ -3,10 +3,10 @@
         {
             "tag": "callabike-aschaffenburg",
             "meta": {
-                "latitude": 49.436467,
```

Comment:
@piotrowski  @eskerda Hi, I'm from the IT of Deutsche Bahn Connect who's the operator of Call a Bike.  
Feel free to use our Open Data API diretly! That is probably way easier than parsing our website :)    
http://data.deutschebahn.com/dataset/flinkster-api

Have fun and feel free to contact me for further questions!